telegram/ext/filters.py

```diff
@@ -153,7 +154,7 @@ class BaseFilter:
     If you want to create your own filters create a class inheriting from either
     :class:`MessageFilter` or :class:`UpdateFilter` and implement a ``filter()``
     method that returns a boolean: :obj:`True` if the message should be
-    handled, :obj:`False` otherwise.
+    handled, otherwise :obj:`False`.
```

Comment:
I think the previous one made more sense

Reply:
✅ Done